config/initializers/flipper.rb

```diff
@@ -12,13 +12,10 @@
 
 Flipper.configure do |config|
   config.default do
-    activerecord_adapter = Flipper::Adapters::ActiveRecord.new
-    cache = ActiveSupport::Cache::MemoryStore.new
-    # Flipper settings will be stored in postgres and cached in memory for 1 minute in production/staging
-    cached_adapter = Flipper::Adapters::ActiveSupportCacheStore.new(activerecord_adapter, cache, expires_in: 1.minute)
-    adapter = Rails.env.development? || Rails.env.test? ? activerecord_adapter : cached_adapter
```

Comment:
In your writeup in the description (which was very useful, by the way), you mention that we use caching in development: `Currently, the vets-api is caching the feature toggle endpoint using ActiveSupport::Cache::MemoryStore(usually implemented in the development environment).`.
However, according to this line, it looks it's actually opposite, we appear to use caching in all environments that aren't `development` or `test`